examples/javascript/DCGAN/DCGAN_Random/index.html

```diff
@@ -6,11 +6,29 @@
 
 	<title>DCGAN Example</title>
 	<script src="http://localhost:8080/ml5.js" type="text/javascript"></script>
-	<script src="sketch.js"></script>
 </head>
 
 <body>
 	<h1>DCGAN Example</h1>
+
+	<canvas width="64" height="64" style="width:200px;"></canvas>
+	<button onclick="generate()" style="display:block;">generate</button>
+
+	<script>
+	let dcgan;
```

Comment:
nit: It looks like the JS code in the `<script>` tag could be tabbed to the right once.